package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "site-natacha",
+  "name": "theodo-uk-meetups",
   "version": "0.1.0",
   "private": true,
   "scripts": {
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint --fix"
```

Comment:
Could you add a separate script to do the fix (sometimes you don't want to lint and fix automatically)?